.github/workflows/pull-request.yml

```diff
@@ -0,0 +1,38 @@
+name: Pull request build
+
+on:
+  pull_request:
+    branches:
+      - dev
```

Comment:
I think we only need to build the PRs sent for the `main` branch as that is the main git branch for a repository.

Reply:
Ok I will remove the dev branch